README.md

```diff
@@ -64,13 +64,18 @@ await transaction.CommitAsync();
 
 ```
 
+## Migrating to v7
+
 ### Sending null values when updating entities.
-When updating entities, Todoist API only updates properties included in the request body, using a `PATCH` request style.
-That's why all properties with `null` values are not included by default, to allow updating without fetching the entity first.
+When updating entities, **Todoist API** only updates properties included in the request body, using a `PATCH` request style.
+That's why all properties with `null` values are not included by default, to allow updating without fetching the entity first,
+since including `null` properties will update them to `null`.
 
-So, if you want to send a `null` value to the API, you need to use the `Unset` extension method.
+- Up until **version 6**, properties like `DueDate` where always included the request body even if set to `null`, to allow users to reset due dates.
+- Starting from **version 7**, however, if you want to send a `null` value to the API, you need to use the `Unset` extension method,
+which could be used with all properties, not only `DueDate`s.
```

Comment:
@olsh 
It's a little bit confusing for me to find a proper way to explain this new change, your opinion\contribution here would be very helpful :)

Reply:
It's always a good idea to include a piece of code with comments to show how the Unset method works.
Something like

```csharp
// create an item with due date
...

// unset the value and update the item
...
```

Reply:
This is what I added, what do you think?

> For example, the following code will not have effect starting from **v7** ❌
> ```csharp
> // This code won't have an effect.
> var task = new UpdateItem("TASK_ID");
> task.DueDate = null;
> 
> await client.Items.UpdateAsync(task);
> ```
> 
> However, using `Unset` will send a `due` property with a `null` value ✔️
> ```csharp
> // This code removes a task's due date.
> var task = new UpdateItem("TASK_ID");
> task.Unset(t => t.DueDate);
> 
> await client.Items.UpdateAsync(task);
> ```

Reply:
Ouch, I missed that :)

I think there is no need to give an example for the previous version. We aren't going to support it anyway.
The example for the new version is pretty much self-explanatory. 👍🏻

Reply:
Ok, should I then remove the `## Migrating to v7` header and comments about migration, or leave it?

Reply:
Yes, I think we can remove the comments too.

Reply:
Done 👍